auctions/templates/claim_status.html

```diff
@@ -38,12 +38,14 @@
                 {% endif %}
             {% endif %}
         </div>
-        <div class="medium-2 columns">
-            <div class="date">
-                <p class="date-label">Pays out</p>
-                <p class="date-details">{{ claim.expires|date:"M" }}<br/>{{ claim.expires|date:"j" }}</p>
-            </div>
-        </div>
+            {% if claim.status != 'Paid' %}
```

Comment:
Might need to move the `if` logic inside the `<div>` element so the grid is always filled.

Have we spot-checked with a claim that has `status == 'Paid'`?